hack/devtools/local_dev_env.sh

```diff
@@ -251,20 +257,15 @@ create_miwi_env_file() {
     mockCert=$(get_mock_msi_cert "$mockMSI")
     mockObjectID=$(get_mock_msi_objectID "$mockClientID")
 
-    setup_platform_identity
```

Comment:
Why remove this step?

Reply:
This step is handled in the go code [here](https://github.com/Azure/ARO-RP/blob/db9f2241faf333ec890ecc19dc8fd57d2819fb56/pkg/util/cluster/cluster.go#L352) and is currently run as part of the cluster creation.

Reply:
Done!